src/backend/distributed/sql/udfs/citus_conninfo_cache_invalidate/latest.sql

```diff
@@ -0,0 +1,9 @@
+ALTER FUNCTION master_conninfo_cache_invalidate()
+RENAME TO citus_conninfo_cache_invalidate;
```

Comment:
It's a bit odd to have these renames in the function definitions (e.g. do we remove it from latest.sql next time we make a change?). 

Maybe we keep the rename in the upgrade scripts?